js/main.js

```diff
@@ -1,29 +1,93 @@
 /* global data */
 /* exported data */
 
-var $form = document.querySelector('#entry-form');
+var $form = document.querySelector('form');
 var $title = document.querySelector('.title');
 var $photoUrl = document.querySelector('.photo-url');
 var $notes = document.querySelector('.notes');
 var $photoUpdate = document.querySelector('.photo');
+var $entriesTab = document.querySelector('a');
+var $p = document.querySelector('p');
+var $ul = document.querySelector('ul');
+var $newButton = document.querySelector('.new-button');
+var $views = document.querySelectorAll('.view');
 
-function photoUpdate(event) {
+$photoUrl.addEventListener('input', function (event) {
   $photoUpdate.setAttribute('src', event.target.value);
-}
+});
 
-function checkSubmit(event) {
+$form.addEventListener('submit', function (event) {
   event.preventDefault();
   var submission = {
     title: $title.value,
     photoUrl: $photoUrl.value,
     notes: $notes.value,
     entryId: data.nextEntryId
   };
+  submission.entryId = data.nextEntryId;
```

Comment:
This line can be removed, you are already doing this within the object literal above